playbooks/bootstrap.yml

```diff
@@ -1,6 +1,44 @@
 ---
-- import_playbook: "./bootstrap_tower.yml"
-- import_playbook: "./bootstrap_integreatly.yml"
-- import_playbook: "./bootstrap_cluster_create.yml"
-- import_playbook: "./bootstrap_cluster_teardown.yml"
-- import_playbook: "./bootstrap_validation.yml"
\ No newline at end of file
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - fail:
```

Comment:
Not sure about this block. The `integreatly_version` variable will always be defined given that it's listed in the integreatly role: https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/defaults/main.yml#L11

What we could do is have a warning prompt if the tower-config project branch (`tower_configuration_project_scm_branch `) and credential project branch (`tower_credentials_project_scm_branch `) differ as this could lead to compatibility issues. We can also remove the integreatly_version variable altogether and have the value of the two project branch variables be set to `master` OR integreatly_version if defined. Hope that makes sense?